pkg/api/dtos/plugins.go

```diff
@@ -6,18 +6,19 @@ import (
 )
 
 type PluginSetting struct {
-	Name          string                 `json:"name"`
-	Type          string                 `json:"type"`
-	Id            string                 `json:"id"`
-	Enabled       bool                   `json:"enabled"`
-	Pinned        bool                   `json:"pinned"`
-	Module        string                 `json:"module"`
-	BaseUrl       string                 `json:"baseUrl"`
-	Info          plugins.Info           `json:"info"`
-	Includes      []*plugins.Includes    `json:"includes"`
-	Dependencies  plugins.Dependencies   `json:"dependencies"`
-	JsonData      map[string]interface{} `json:"jsonData"`
-	DefaultNavUrl string                 `json:"defaultNavUrl"`
+	Name             string                 `json:"name"`
+	Type             string                 `json:"type"`
+	Id               string                 `json:"id"`
+	Enabled          bool                   `json:"enabled"`
+	Pinned           bool                   `json:"pinned"`
+	Module           string                 `json:"module"`
+	BaseUrl          string                 `json:"baseUrl"`
+	Info             plugins.Info           `json:"info"`
+	Includes         []*plugins.Includes    `json:"includes"`
+	Dependencies     plugins.Dependencies   `json:"dependencies"`
+	JsonData         map[string]interface{} `json:"jsonData"`
+	SecureJsonFields map[string]bool        `json:"secureJsonFields"`
```

Comment:
Is this a real change, or just adding/using type where we currently use `map[string]...`?

I think it is the latter, but correct me if I am wrong

Reply:
It is a real change. We are adding the `SecureJsonFields` so we can detect what parts of secureJsonData that has been configured.

Reply:
A real change in the sense that we're passing back `SecureJsonFields` as part of the DTO now similar to datasources

Reply:
This will only be passed back for app plugins, right?